lib/grandpa/grandpa.go

```diff
@@ -483,6 +483,7 @@ func (s *Service) playGrandpaRound() error {
 	go s.sendVoteMessage(prevote, vm, roundComplete)
 
 	logger.Debug("receiving pre-commit messages...")
+	// through goroutine s.receiveMessages(ctx)
```

Comment:
Maybe add this to the debug line, since it's for 'debug'

```suggestion
	logger.Debug("receiving pre-commit messages from goroutine s.receiveMessages...")
```

Reply:
better to not put function names in log messages